### DISCUSSION TITLE
Zyxel Keenetic 4G III

### DISCUSSION THREAD

**Discussion by samorez1** [qzeleza/kvas#99] [General]
Подскажите, пожалуйста, запустится kvas на старом Zyxel Keenetic 4G III?

**Comment by Nicolayka** [comment]
> Подскажите, пожалуйста, запустится kvas на старом Zyxel Keenetic 4G III?

Ответ кроется [здесь](https://help.keenetic.com/hc/ru/articles/360021214160-%D0%A3%D1%81%D1%82%D0%B0%D0%BD%D0%BE%D0%B2%D0%BA%D0%B0-%D1%81%D0%B8%D1%81%D1%82%D0%B5%D0%BC%D1%8B-%D0%BF%D0%B0%D0%BA%D0%B5%D1%82%D0%BE%D0%B2-%D1%80%D0%B5%D0%BF%D0%BE%D0%B7%D0%B8%D1%82%D0%BE%D1%80%D0%B8%D1%8F-Entware-%D0%BD%D0%B0-USB-%D0%BD%D0%B0%D0%BA%D0%BE%D0%BF%D0%B8%D1%82%D0%B5%D0%BB%D1%8C)

> Важно! Установка OPKG-пакетов возможна на моделях с USB-портами, c поддержкой работы USB-накопителей. Это актуальные модели 4G (KN-1212), Omni (KN-1410), Extra (KN-1710/1711/1713), Giga (KN-1010/1011), Ultra SE (KN-2510), Giga SE (KN-2410), Ultra (KN-1810/1811), Viva (KN-1910/1912), Hero 4G (KN-2310), DSL (KN-2010), Duo (KN-2110), Ultra SE (KN-2510), Giant (KN-2610), Peak (KN-2710), Hopper (KN-3810), Hopper DSL (KN-3610) и модели прошлых поколений Zyxel Keenetic II / III, Extra, Extra II, Giga II / III, Omni, Omni II, Viva, Ultra, Ultra II.

Окончательный ответ - да!
